Formula/g/gcc@13.rb

```diff
@@ -120,7 +122,8 @@ def install
       # To make sure GCC does not record cellar paths, we configure it with
       # opt_prefix as the prefix. Then we use DESTDIR to install into a
       # temporary location, then move into the cellar path.
-      system "make", install_target, "DESTDIR=#{Pathname.pwd}/../instdir"
+      # Deparallelize for modula 2 as make install will block indefinitely.
+      ENV.deparallelize { system "make", install_target, "DESTDIR=#{Pathname.pwd}/../instdir" }
```

Comment:
Has this been reported upstream?

Reply:
Modula 2 requires flex for building. Flex comes with MacOS, but not with the Ubuntu CI container.

Your "instead" puzzles me. Don't you mean "as well"? I would assume that this is the way to go (and that is what I just pushed):

```
  uses_from_macos "flex"

  on_linux do
    ...
    depends_on "flex"
  end
```